build.sh

```diff
@@ -57,6 +57,7 @@ res_540p=( "Performance" 960 540 )
 res_720p219=( "Performance" 1706 720 )
 res_720p489=( "Performance" 3840 720 )
 res_720p=( "Performance" 1280 720 )
+res_768p=( "Performance" 1366 768 )
```

Comment:
I'd probably put 768p under Quality since it's higher than 720p, which is what most Wii U games run at.